inductiva/fluids/wind_tunnel/wind_tunnel.py

```diff
@@ -122,73 +121,20 @@ def simulate(
             raise ValueError("WindTunnel is empty. Object path not specified.")
 
         self.object_path = utils.files.resolve_path(object_path)
-        self.num_iterations = num_iterations
-        self.resolution = MeshResolution[resolution.upper()].value
+        self.params["num_iterations"] = num_iterations
+        self.params["resolution"] = MeshResolution[resolution.upper()].value
 
         commands = self.get_commands()
-
         task = super().simulate(simulator,
                                 machine_group=machine_group,
                                 storage_dir=storage_dir,
                                 commands=commands)
 
         return task
 
-    def get_commands(self):
-        """Returns the commands for the simulation."""
-
-        commands_file_path = os.path.join(SCENARIO_TEMPLATE_DIR,
-                                          OPENFOAM_TEMPLATE_INPUT_DIR,
-                                          COMMANDS_FILE_NAME)
-
-        commands = self.read_commands_from_file(commands_file_path)
-
-        return commands
+    def add_extra_input_files(self, simulator: simulators.Simulator, input_dir):  # pylint: disable=unused-argument
```

Comment:
The name of the method is not self-explanatory, can you be more specific like what extra files?

Reply:
It depends of the simulator and what users want to "parameterize"... That is why I didn't make more expliciti..